eqllib/analytics/execution/T1170-execution-of-stager-via-commonly-abused-executable.toml

```diff
@@ -0,0 +1,20 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies the execution of and subsequent network connection from a commonly abused Windows executable. This behavior is indicative of a stager being executed on an endpoint and a network connection being established to download additional malicious code."
+id = "2e6b3e3f-154d-4c26-ba5b-eacfc7f1a0ef"
+name = "Execution of Stager via Commonly Abused Executable"
+os = ["windows"]
+tactics = ["Execution", "Defense Evasion"]
+tags = ["koadic"]
+techniques = ["T1170", "T1117", "T1085", "T1047"]
+updated_date = "12/04/2019"
+
+[analytic]
+query = '''
+sequence by unique_pid
+  [process where subtype.create and process_name in ("mshta.exe", "regsvr32.exe", "rundll32.exe", "wmic.exe")]
```

Comment:
We already have detections for some of these, so I think this rule is redundant.

- [Mshta Network Connections](https://eqllib.readthedocs.io/en/latest/analytics/6bc283c4-21f2-4aed-a05c-a9a3ffa95dd4.html)
- [Remote Execution via WMIC](https://eqllib.readthedocs.io/en/latest/analytics/07b1481c-2a20-4274-a64e-effcd40941a5.html)

For regsvr32.exe, we should publish the Endgame rule "Registration Tool Making Network Connections".

Rundll32.exe can actually be pretty noisy. But I think my proposal to the earlier RunDLL32 rule should address that.

Reply:
Yeh I am fine with leaving this off.  It will be shared in the blog post and they can use that example.